.changeset/dull-zoos-sell.md

```diff
@@ -0,0 +1,7 @@
+---
+"@aptos-labs/wallet-adapter-ant-design": minor
```

Comment:
if there is no way for end users to disable those screens, we might want to go with a major bump

Reply:
Currently, Aptos Connect can be disabled by not passing it to the `optInWallets` prop on the `AptosWalletProvider` component — this would also hide the education screens. I think we generally want projects to always display the education screens when Aptos Connect is available, but let me know what you think.

Reply:
> Currently, Aptos Connect can be disabled by not passing it to the `optInWallets` prop on the `AptosWalletProvider` component — this would also hide the education screens. I think we generally want projects to always display the education screens when Aptos Connect is available, but let me know what you think.

actually, for now, [we always include AptosConnect](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/src/WalletCore.ts#L285).

I agree we probably want projects to always display it, but it should be their decision eventually, and they should be able to simply do it by upgrading the react package

Reply:
Okay, I'll make this a major version bump then.

Reply:
I updated the changeset